arch/arm/boot/dts/bcm2710-rpi-cm3.dts

```diff
@@ -6,6 +6,7 @@
 #include "bcm283x-rpi-csi1-4lane.dtsi"
 
 / {
+	compatible = "raspberrypi,3-compute-module", "brcm,bcm2837";
 	model = "Raspberry Pi Compute Module 3";
```

Comment:
Seems odd to start with the 3, rather than compute-module-3. Any reason for that?

Reply:
Looking at Documentation/devicetree/bindings/arm/bcm/brcm,bcm2835.txt shows it's the right compatible.

Reply:
https://github.com/raspberrypi/linux/blob/rpi-4.19.y/arch/arm/boot/dts/bcm2709-rpi-2-b.dts#L9
https://github.com/raspberrypi/linux/blob/rpi-4.19.y/arch/arm/boot/dts/bcm2710-rpi-3-b.dts#L9
https://github.com/raspberrypi/linux/blob/rpi-4.19.y/arch/arm/boot/dts/bcm2710-rpi-3-b-plus.dts#L9